main.tf

```diff
@@ -225,6 +225,17 @@ module "kubeconfig" {
   depends_on = [module.eks.cluster_name] # The name/id of the EKS cluster. Will block on cluster creation until the cluster is really ready.
 }
 
+# Normally, the use of local-exec below is avoided. It is used here to patch the gp2 storage class as the default storage class for EKS 1.30 and later clusters.
+# In the near future, adopting a newer version of the aws-ebs-csi-driver will create a new gp3-based storage class which will become the default storage class.
```

Comment:
Can we go ahead and create that ticket, then, add to this explanation for posterity?

Reply:
Sure, @undr-rowr, will be pushing up a new commit with that ticket reference shortly.

Reply:
@undr-rowr, s/b ready for re-review

Reply:
Done, see the latest commit for change.